design/pv_backup_info.md

```diff
@@ -42,37 +42,39 @@ The 1 version of `VolumeInfo` definition is:
 type VolumeInfoV1 struct {
     PVCName        string    // The PVC's name. The format should be <namespace-name>/<PVC-name>
     PVName         string    // The PV name.
-    BackupMethod   string    // The way the volume data is backed up. The valid value includes `VeleroNativeSnapshot`, `PodVolumeBackup`, `CSISnapshot` and `Skipped`.
-    SnapshotDataMovement bool // Whether the volume's snapshot data is moved to specified storage.
+    BackupMethod   string    // The way the volume data is backed up. The valid value includes `VeleroNativeSnapshot`, `PodVolumeBackup` and `CSISnapshot`.
+    SnapshotDataMoved bool   // Whether the volume's snapshot data is moved to specified storage.
 
+    Skipped         boolean   // Whether the Volume is skipped in this backup.
     SkippedReason   string    // The reason for the volume is skipped in the backup.
     StartTimestamp  *metav1.Time // Snapshot starts timestamp.
 
     CSISnapshotInfo          CSISnapshotInfo
-    SnapshotDataMoveInfo     SnapshotDataMoveInfo
+    SnapshotDataMovementInfo SnapshotDataMovementInfo
     NativeSnapshotInfo       VeleroNativeSnapshotInfo
     PVBInfo                  PodVolumeBackupInfo
+    PVInfo                   PVInfo
 }
 
 // CSISnapshotInfo is used for displaying the CSI snapshot status
 type CSISnapshotInfo struct {
-    SnapshotHandle  string       // The actual snapshot ID. It can be the cloud provider's snapshot ID or the file-system uploader's snapshot.
+    SnapshotHandle  string       // It's the storage provider's snapshot ID for CSI.
     Size            int64        // The snapshot corresponding volume size. Some of the volume backup methods cannot retrieve the data by current design, for example, the Velero native snapshot.
 
     Driver          string  // The name of the CSI driver.
     VSCName         string // The name of the VolumeSnapshotContent. 
 }
 
-// SnapshotDataMoveInfo is used for displaying the snapshot data mover status.
-type SnapshotDataMoveInfo struct {
+// SnapshotDataMovementInfo is used for displaying the snapshot data mover status.
+type SnapshotDataMovementInfo struct {
```

Comment:
A repo snapshot ID is also required for data mover

Reply:
Added.